php/simps/composer.json

```diff
@@ -1,11 +1,13 @@
 {
   "require": {
-    "simple-swoole/simps": "dev-master",
-    "swoole/library": "dev-master"
+    "simple-swoole/simps": "dev-master"
   },
   "autoload": {
     "psr-4": {
       "App\\": "app/"
     }
+  },
+  "config": {
+    "optimize-autoloader": true
```

Comment:
what does it mean ?

is it related to `composer dumpautoload -o`

Reply:
Yes, it is. Is there anything wrong?

Reply:
I saw it in the dockerfile, let me delete it